openfe/protocols/openmm_rbfe/_rbfe_utils/topologyhelpers.py

```diff
@@ -374,8 +374,8 @@ def set_and_check_new_positions(mapping, old_topology, new_topology,
         Positions of the alchemically changing molecule in the "new" alchemical
         state.
     tolerance : float
-        Maximum allowed deviation along any dimension (x,y,z) in mapped atoms
-        between the "old" and "new" positions. Default 0.5.
+        Warning threshold for deviations along any dimension (x,y,z) in mapped
```

Comment:
Keeping this in because it's a nice diagnostic, but am completely removing it as a user option.

Reply:
I think this is another thing I'd want to put under the "what do our safety rails do" category